store/postgres/src/store_events.rs

```diff
@@ -0,0 +1,65 @@
+use graph::prelude::*;
+use graph::serde_json;
+use notification_listener::{NotificationListener, SafeChannelName};
+use store::StoreEvent;
+
+pub struct StoreEventListener {
+    notification_listener: NotificationListener,
+}
+
+impl StoreEventListener {
+    pub fn new(postgres_url: String) -> Self {
+        StoreEventListener {
+            notification_listener: NotificationListener::new(
+                postgres_url,
+                SafeChannelName::i_promise_this_is_safe("entity_changes"),
```

Comment:
I was going to suggest changing the channel name but it's not worth it. The migration required to drop existing legs fun procedures and replace them just to call this "store_events" would be to complex.